pages/upgrade/graylog-4.1.rst

```diff
@@ -13,6 +13,11 @@ Upgrading to Graylog 4.1.x
 Breaking Changes
 ================
 
+The limit parameter in the legacy search api (``/search/universal/(absolute|keyword|relative)``) semantics has changed
```

Comment:
I overlooked this in the core repo PR, but it should be:
```suggestion
The limit parameter in the legacy search API (``/search/universal/(absolute|keyword|relative)``) semantics have changed
```

I would even suggest turning the sentence around:
```suggestion
The semantics of the limit parameter in the legacy search API (``/search/universal/(absolute|keyword|relative)``) have changed
```